src/commands/install-composer.yml

```diff
@@ -0,0 +1,7 @@
+description: "Install Composer"
+steps:
+  - run:
+      name: "Install Composer"
+      command: |
+        curl -sS https://getcomposer.org/installer -o composer-setup.php
+        sudo php composer-setup.php --install-dir=/usr/local/bin --filename=composer
```

Comment:
I feel like if we're going to define an install directory for composer, we could probably make this a configurable parameter, aligning with how other Orbs handle installation (e.g circleci-public/UPX-orb).

Reply:
Added `force` and `install-dir` params in https://github.com/CircleCI-Public/php-orb/pull/25/commits/0a97e4084fa078df9a34294e6f7df652514a4f10